CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_zendesk v0.14.0
+
+[PR #136](https://github.com/fivetran/dbt_zendesk/pull/136) includes the following changes:
+
+## Bug Fixes
+- Converted the `zendesk__sla_policies` metric for `sla_elapsed_time` to be reported in minutes to the second as opposed to just the nearest rounded minute. This ensures more accurate reporting.
```

Comment:
I noticed we still use minutes as the date diff grain in [sla_policies](https://github.com/fivetran/dbt_zendesk/blob/ebb1c84e8ecb67da1dd60c276f608971ba6f129e/models/zendesk__sla_policies.sql#L126). Should we just standardize everything to second? There are a few other places where we still use minutes as well.

Reply:
Ah yeah good catch, can update that to be the same as the other calculations

Reply:
Btw I saw minutes being used in a few other date diff calcs as well

Reply:
Updated the other relevant `minutes` calculations to be `seconds` within the final `zendesk__sla_policies` model as well as within the `int_zendesk__reply_time_calendar_hours` model.

Some date diff calcs that aren't ultimately used to calculate the elapsed time metric don't need to use `seconds`. For example there are a few calculations within the `int_zendesk__reply_time_business_hours` model that still use `minutes` as they are calculating the schedule overlaps for minutes from Sunday. These aren't used in the calculation for elapsed time and probably could lead to more issues if we did use `seconds` for these calculations. It is probably best for these to remain `minutes`.